docs/jazzy.yml

```diff
@@ -22,68 +22,108 @@ custom_categories:
       - RouteProgress
       - RouteLegProgress
       - RouteStepProgress
-      - RouteControllerNotificationUserInfoKey
+      - NavigationEventsManager
+      - EventsManagerDataSource
+      - NavigationLocationManager
+      - ReplayLocationManager
+      - SimulatedLocationManager
+      - NavigationRouteOptions
+      - NavigationMatchOptions
+      - NavigationDirections
+      - SimulationMode
   - name: Maps
     children:
       - NavigationMapView
       - NavigationMapViewDelegate
       - MGLMapView
   - name: Styling
     children:
+      - LanesStyleKit
+      - ManeuversStyleKit
       - Style
-      - StyleKitArrows
-      - StylableButton
       - StylableLabel
+      - StylableView
+      - StylableTextView
+      - StylableButton
+      - StyleManager
+      - StyleManagerNotificationUserInfoKey
+      - StyleManagerDelegate
+      - DayStyle
+      - NightStyle
   - name: Formatters
     children:
       - DistanceFormatter
-      - RouteStepFormatter
   - name: UI Components
     children:
-      - RouteTableViewController
-      - NavigationPulleyViewController
       - Button
-      - CellSubtitleLabel
-      - CellTitleLabel
       - DashedLineView
       - HighlightedButton
       - LineView
       - ProgressBar
       - SeparatorView
       - SubtitleLabel
       - TitleLabel
-      - TurnArrowView
+      - ManeuverView
       - WayNameLabel
+      - BottomBannerViewController
+      - BottomBannerViewControllerDelegate
+      - SpeedLimitView
+      - UserPuckCourseView
+      - CourseUpdatable
+      - GenericRouteShield
+      - NavigationComponent
   - name: CarPlay
     children:
       - CarPlayManager
       - CarPlayManagerDelegate
       - CarPlayNavigationViewController
       - CarPlayNavigationDelegate
       - CarPlayActivity
+      - CarPlayCompassView
+      - CarPlayMapViewController
+      - CarPlaySearchController
+      - CarPlayConnectionObserver
   - name: Configuration
     children:
-      - MaxSecondsSpentTravelingAwayFromStartOfRoute
       - RouteControllerDeadReckoningTimeInterval
+      - RouteControllerMediumAlertInterval
       - RouteControllerHighAlertInterval
       - RouteControllerManeuverZoneRadius
-      - RouteControllerMaxManipulatedCourseAngle
+      - RouteSnappingMaxManipulatedCourseAngle
       - RouteControllerMaximumAllowedDegreeOffsetForTurnCompletion
       - RouteControllerMaximumDistanceBeforeRecalculating
-      - RouteControllerMediumAlertInterval
-      - RouteControllerMinimumDistanceForHighAlertCycling
-      - RouteControllerMinimumDistanceForHighAlertDriving
-      - RouteControllerMinimumDistanceForHighAlertWalking
-      - RouteControllerMinimumDistanceForMediumAlertCycling
-      - RouteControllerMinimumDistanceForMediumAlertDriving
-      - RouteControllerMinimumDistanceForMediumAlertWalking
       - RouteControllerUserLocationSnappingDistance
+      - MBCongestionAttribute
+      - MBCurrentLegAttribute
+      - MBRouteLineWidthByZoomLevel
```

Comment:
Somehow we missed these constants when removing the class prefix from global symbols. As a separate task, we should rename these symbols to remove the class prefix. To avoid conflicts with other modules, we should nest them inside the class that uses them, NavigationMapView.

/cc @JThramer